### PR TITLE
Add SetOrg intermediary component to set org

### PR DESCRIPTION
**Pull request by ebb-tide**
Closes #

_Briefly describe your proposed changes:_

  - [ ] CHANGELOG.md updated with a link to the PR (not the Issue)
  - [ ] Rebased/mergeable
  - [ ] Tests pass
  - [ ] http/swagger.yml updated (if modified Go structs or API)
  - [ ] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: ui/src/shared/containers/SetOrg.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+}
+
+interface StateProps {
+  orgs: Organization[]
+  status: RemoteDataState
+}
+
+type Props = StateProps & DispatchProps & PassedInProps
+
+const GetOrganizations: FunctionComponent<Props> = ({
```
Comment (121watts) [inline ui/src/shared/containers/SetOrg.tsx]: Looks like some copy pasta.  `SetOrg`?

File: ui/src/shared/containers/SetOrg.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      setOrg(foundOrg)
+      return
+    }
+
+    // else default to first org
+    router.push(`orgs/${orgs[0].id}`)
+  }, [orgID])
+
+  return (
+    <SpinnerContainer loading={status} spinnerComponent={<TechnoSpinner />}>
```
Comment (121watts) [inline ui/src/shared/containers/SetOrg.tsx]: No asynchronous work is happening here.  I don't think any of the loading or spinning states / logic / presentation is needed.

File: ui/src/shared/containers/SetOrg.tsx (hunk 1 above)
Comment (ebb-tide) [reply ui/src/shared/containers/SetOrg.tsx]: thanks!

File: ui/src/shared/containers/SetOrg.tsx (hunk 2 above)
Comment (ebb-tide) [reply ui/src/shared/containers/SetOrg.tsx]: actually, we don't want to render the children unless org has been set.

**Pull request merged**